src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -17,6 +17,7 @@ public partial class HttpClientHandler : HttpMessageHandler
         private readonly CurlHandler _curlHandler;
         private readonly SocketsHttpHandler _socketsHttpHandler;
         private readonly DiagnosticsHandler _diagnosticsHandler;
+        private ClientCertificateOption _clientCertificateOptions;
```

Comment:
If we wanted to, we could avoid this by caching the delegate used for automatic in a static readonly field, and then use reference comparison on the delegate stored in _socketsHttpHandler.SslOptions.LocalCertificateSelectionCallback to determine which mode was set, Automatic if the reference matches and Manual if it doesn't.  I'm ok with just having this extra field for clarity, though.  Either way.

Reply:
I'll leave it as-is for clarity.